config/initializers/monkey_patching.rb

```diff
@@ -0,0 +1,21 @@
+module Arel
```

Comment:
j'ai lu récemment sur les bonnes pratiques de monkey patching en rails. 
https://www.justinweiss.com/articles/3-ways-to-monkey-patch-without-making-a-mess/

Personnellement j'aime bien la dernière solution qui s'appelle 

> My all-time favorite monkey patch

Reply:
J'ai tenté de faire cela (@morissetcl peut en témoigner) mais j'ai échoué. Je veux bien retenter avec toi.

Reply:
ok je n'y arrive pas non plus :/